### PR TITLE
Entity renderers

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Changes:-
- Uncursed an inventory
- Uncursed entity renderers and ocelots
- Uncursed villagers and opengl

no changelog becoz vi doesn't have pasting

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/BipedEntityRenderer.mapping [hunk 1]
```diff
@@ -1 +1,9 @@
 CLASS net/minecraft/class_1147 net/minecraft/client/render/entity/BipedEntityRenderer
+	FIELD field_4863 bipedModel Lnet/minecraft/class_866;
```
Comment (hYdos) [inline mappings/net/minecraft/client/render/entity/BipedEntityRenderer.mapping:2]: does yarn use the Biped name?

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/render/entity/BipedEntityRenderer.mapping:2]: the `model` field is already declared. so i prefixed each with the class prefix

File: mappings/net/minecraft/client/render/entity/SheepEntityRenderer.mapping [hunk 2]
```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_1164 net/minecraft/client/render/entity/SheepEntityRenderer
+	FIELD field_4920 SHEEP_TEX Lnet/minecraft/class_1605;
```
Comment (hYdos) [inline mappings/net/minecraft/client/render/entity/SheepEntityRenderer.mapping:2]: TEX in static doesnt look good

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/render/entity/SheepEntityRenderer.mapping:2]: yarn does it

**Comment by BoogieMonster1O1** [comment]
ok lemme merge

**Comment by hYdos** [comment]
wait for shedaniel

**Comment by BoogieMonster1O1** [comment]
no

**Comment by BoogieMonster1O1** [comment]
pls no

**Comment by hYdos** [comment]
its happening

**Comment by BoogieMonster1O1** [comment]
ok i'm merging

**Comment by hYdos** [comment]
no

**Comment by BoogieMonster1O1** [comment]
why

**Comment by hYdos** [comment]
multiple opinions are important

**Pull request merged**